install-dev.sh

```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
+
+# Check if pip is installed, and install it if necessary
+if ! command -v pip >/dev/null 2>&1; then
+    echo "---- Installing pip ----"
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+    python get-pip.py
+else
+    echo "---- Pip is already installed ----"
+fi
+
+
 echo "---- Installing Python Poetry ----"
 pip install -U pip
```

Comment:
You could move that line up and say "updating pip"

Reply:
Do you want me to remove the check which was done by the if statement?